### PR TITLE
Tokens and Nfts - Updated token-program-advanced lesson 

### PR DESCRIPTION
**Pull request by ubadineke**
### Summary of Changes
- Updated solana's client from web3 to web3.js
- Added top level async function
- Added example explorer transaction link for all lab exercises.
- Separated lab exercises(revoke and burn) into different files.

Fixes #

### CODE REVIEW

File: content/courses/tokens-and-nfts/token-program-advanced.md [hunk 1]
```diff
@@ -210,59 +210,109 @@ const delegate = new PublicKey("YOUR_DELEGATE_HERE");
...
 const tokenMintAccount = new PublicKey("YOUR_TOKEN_MINT_ADDRESS_HERE");
 
-// Get or create the source and destination token accounts to store this token
-const sourceTokenAccount = await getOrCreateAssociatedTokenAccount(
-  connection,
-  user,
-  tokenMintAccount,
-  user.publicKey,
-);
+(async () => {
```
Comment (mikemaccana) [inline content/courses/tokens-and-nfts/token-program-advanced.md:213]: You don't need to add this, in fact you were instructed to remove it: https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#jsts

Also add `filesname=` to the code snippets. See https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#file-names

**Review by mikemaccana — CHANGES_REQUESTED** [review]
See comments above.

**Comment by ubadineke** [comment]
@mikemaccana I've implemented the changes.
check it out.

**Comment by ubadineke** [comment]
Hi @mikemaccana 
Are there any other changes you want me to implement.

**Review by mikemaccana — APPROVED** [review]

**Comment by mikemaccana** [comment]
@nickfrosty this looks good, can you please merge it?

**Pull request closed without merging**